cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/CruiseControlApiImpl.java

```diff
@@ -67,6 +71,7 @@ public CruiseControlApiImpl(int idleTimeout, Secret ccSecret, Secret ccApiSecret
         this.authHttpHeader = getAuthHttpHeader(apiAuthEnabled, ccApiSecret);
         this.pemTrustSet = new PemTrustSet(ccSecret);
         this.httpClient = buildHttpClient();
+        this.objectMapper = new ObjectMapper();
```

Comment:
Wouldn't a static `ObjectMapper` instance as you did for the `KafkaRebalanceAssemblyOperator` be better?